### PR TITLE
added callout on tmux to screen lesson

### PR DESCRIPTION
**Pull request by goi42**
Besides the fact that I personally prefer tmux (plan to teach the "screen" lesson using it), I think this is good for consistency as the "permanent screen" lesson uses tmux as well.

### CODE REVIEW

File: shell/screen.md [hunk 1]
```diff
@@ -114,3 +114,14 @@ $ screen
...
+`tmux` is another program you can use to keep things running remotely. 
+In practice, it is very similar to screen, but it has some minor 
+differences. You can find syntax guides easily online, including guides 
+showing the equivalent commands in tmux and screen, but here's a quick 
+list of equivalent commands to those used in this lesson:
+* `tmux ls` instead of `screen -list`
+* `Ctrl-b d` to detach instead of `Ctrl-a d` (`tmux` in general uses 
+  `Ctrl-b` instead of `Ctrl-a`)
+* `tmux a` or `tmux attach` instead of `screen -rD`
+{% endcallout %}
```
Comment (alexpearce) [inline shell/screen.md:128]: It would be nice to link to a guide or to here (because we might be in a better position to judge what guides are good).

**Comment by alexpearce** [comment]
Good idea! I'll let you decide whether it's worth adding links to guides, then we can merge.

File: shell/screen.md (hunk 1 above)
Comment (goi42) [reply shell/screen.md:128]: Yeah, the reason I didn't was that the guides I use seem to be pretty unofficial. [This one](http://hyperpolyglot.org/multiplexers) is quite useful, but I wasn't sure whether we wanted to link to it.

Comment (alexpearce) [reply shell/screen.md:128]: IMO unofficial is fine. Something styled more like a blog post would be cool, but the one you posted is good as a reference.

Comment (goi42) [reply shell/screen.md:128]: done. (Didn't immediately find a good blog post--stuck with the link above.)

Comment (alexpearce) [reply shell/screen.md:128]: 👍 Thanks a bunch!

**Pull request merged**